kratos/python/add_other_utilities_to_python.cpp

```diff
@@ -828,6 +829,36 @@ void AddOtherUtilitiesToPython(pybind11::module &m)
         .def("CreateSphereTriangularMesh", &CppTestsUtilities::CreateSphereTriangularMesh)
     ;
 
+    m.def_submodule("ModelPartUtils", "Auxiliary utilities for model parts.")
```

Comment:
Its a class, not a namespace, this is wrong

Reply:
Okay